CodeEdit/AppDelegate.swift

```diff
@@ -176,6 +176,10 @@ final class AppDelegate: NSObject, NSApplicationDelegate, ObservableObject {
         FeedbackView().showWindow()
     }
 
+
+    /// Tries to focus a window with specified view content type.
+    /// - Parameter type: The type of viewContent which hosted in a window to be focused.
+    /// - Returns: ``true`` if window exist and focused, oterwise - ``false``
```

Comment:
note that double back-ticks are used to reference properties, methods, types,… for inline code use single back-ticks.